sub-process.c

```diff
@@ -29,12 +29,14 @@ struct subprocess_entry *subprocess_find_entry(struct hashmap *hashmap, const ch
 
 int subprocess_read_status(int fd, struct strbuf *status)
 {
+	char buffer[LARGE_PACKET_MAX];
```

Comment:
I'm rebasing onto 2.27 and going to try omitting all the pkt-line changes.
Since I didn't actually multi-thread the foreground process and always do
sync conversations with the pool of background processes, the foreground
process won't have concurrent conversations.  So, I think I can skip all the _r
changes for another effort.